### ISSUE TITLE
Newman can't parse the http verb dynamicaly

### ISSUE THREAD

**Issue by lsousa1988** [postmanlabs/newman#3255]
Hi everyone,
I'm facing a problem and probably I'm the problem but I'll give a try.

Newman cli version: 6.1.3
OS: ubuntu

I have two files, 1 postman collection and 2 csv file with verb,endpoint, payload

I'm running this command - "run .\postman_collection.json --iteration-data .\mock_data.csv"
and I'm stuck in this error - 
" 
#  failure                detail
 1.  TypeError              Method must be a valid HTTP token ["{{METHOD}}"]
                            at request
                            inside "
"

[mock_data.csv](https://github.com/user-attachments/files/16353421/mock_data.csv)
[postman_collection.json](https://github.com/user-attachments/files/16353422/postman_collection.json)

Long story short, I'm trying to create a pipeline in azure devops to test some endpoints with differents HTTP verbs.

I appreciate your help.

Regards.
